src/modules/keys.js

```diff
@@ -0,0 +1,7 @@
+let keyId = '';
+
+const getKey = () => keyId.toString();
+
+const setKey = (id) => { keyId = id; };
+
+export { setKey, getKey };
```

Comment:
:red_circle: It is important to keep your code very simple. When you add complexity to your code, you are increasing the number of opportunities for errors to occur. You are also making it more difficult to understand, debug, and maintain your code. In most cases, the benefits of simplicity outweigh the benefits of complexity.

Knowing that the gameID is a static value that will be generated only once You can just keep your KeyId variable and export it There is no place for setters and getters here. 
What do you think if your store your gameID in a constant that you will export? You can follow this example


```
export const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
export const KeyId = 'your-game-id-here'; // Example: 'GgbsQ7kNrXgkbGP6O5B2'
```

Reply:
> Correction 

The key has been put in a constant that will later be imported for use in the urls.